pkg/operator/operator.go

```diff
@@ -279,11 +279,25 @@ func (optr *Operator) sync(key string) error {
 		return err
 	}
 
+	// sync up keys and certs
+	etcdMetricsServerCert, err := optr.getKeyFromSecret("kube-system", "etcd-metrics-server", "tls.crt")
```

Comment:
this is now failing here cause the secret isn't in the cluster

From CI:

> level=fatal msg="failed to initialize the cluster: Cluster operator machine-config is reporting a failure: Failed to resync 0.0.1-2019-03-08-124327 because: secrets \"etcd-metrics-server\" not found"

is the installer, today, creating those secrets?

Reply:
see below

Reply:
ok, given these secrets aren't in the cluster, that installer PR where you add these secrets must land first (right?) or this cannot pass

Reply:
right but if I add that to code to my custom installer and use this release from CI as below then it should work but I still get 

```